tests/unit/test_upload.py

```
@@ -10,12 +10,30 @@
 from s3_upload.utils import upload
 
 
+@patch("s3_upload.utils.upload.post_slack_message")
 @patch("s3_upload.utils.upload.boto3.Session")
 class TestCheckAwsAccess(unittest.TestCase):
     @patch("s3_upload.utils.upload.AWS_ACCESS_KEY", None)
     @patch("s3_upload.utils.upload.AWS_SECRET_KEY", None)
     @patch("s3_upload.utils.upload.AWS_DEFAULT_PROFILE", "baz")
-    def test_list_of_buckets_returned_on_aws_being_accessible(self, mock_s3):
+    def test_list_of_buckets_returned_on_aws_being_accessible_w_default_profile(
```

Ruff (0.7.0)

Missing return type annotation for public function `test_list_of_buckets_returned_on_aws_being_accessible_w_default_profile`

Add return type annotation: `None`

(ANN201)